### ISSUE TITLE
discovery: document navigation

### ISSUE THREAD

**Issue by mike-winberry** [mike-winberry/oscalot#9]
Description:
- depends on #6 
- once a canvas is established for editing, research should be placed into simplifying documentation navigation, readability, and accessibility with a focus towards engineering oscal developers and maintainers.
